From ef46380e802222681871db9f367025ce414d7ebd Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin2.jansa@lgepartner.com>
Date: Wed, 1 Feb 2023 23:22:42 +0000
Subject: [PATCH] Fix build with gcc-13

:Release Notes:

:Detailed Notes:
Fixes:
http://gecko.lge.com/Errors/Details/529539

Compiling TOPDIR/BUILD/work/qemux86_64-webos-linux/libedgetpu/grouper-r5/git/makefile_build/../tflite/custom_op.cc
In file included from TOPDIR/BUILD/work/qemux86_64-webos-linux/libedgetpu/grouper-r5/git/makefile_build/../driver/run_controller.h:22,
                 from TOPDIR/BUILD/work/qemux86_64-webos-linux/libedgetpu/grouper-r5/git/makefile_build/../driver/run_controller.cc:15:
TOPDIR/BUILD/work/qemux86_64-webos-linux/libedgetpu/grouper-r5/git/makefile_build/../driver/hardware_structures.h:86:3: error: 'uint64_t' does not name a type
   86 |   uint64_t reserved;
      |   ^~~~~~~~
TOPDIR/BUILD/work/qemux86_64-webos-linux/libedgetpu/grouper-r5/git/makefile_build/../driver/hardware_structures.h:25:1: note: 'uint64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
   24 | #include "port/macros.h"
  +++ |+#include <cstdint>
   25 |

:Testing Performed:
Only build tested.

:QA Notes:
No change to image.

:Issues Addressed:
[WRP-8053] CCC: Various build fixes
---
Upstream-Status: Pending

 driver/hardware_structures.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/driver/hardware_structures.h b/driver/hardware_structures.h
index d17c678..64a6a8d 100644
--- a/driver/hardware_structures.h
+++ b/driver/hardware_structures.h
@@ -18,6 +18,7 @@
 #define DARWINN_DRIVER_HARDWARE_STRUCTURES_H_
 
 #include <stddef.h>
+#include <cstdint>
 #include <algorithm>
 
 #include "port/integral_types.h"
